be/src/vec/exec/format/json/new_json_reader.cpp

```diff
@@ -1475,15 +1476,18 @@ Status NewJsonReader::_simdjson_write_data_to_column(simdjson::ondemand::value&
         break;
     }
     default: {
-        auto str_view = simdjson::to_json_string(value).value();
         if (value.type() == simdjson::ondemand::json_type::string) {
+            uint8_t* unescape_buffer =
+                    reinterpret_cast<uint8_t*>(&_simdjson_ondemand_unscape_padding_buffer[0]);
+            std::string_view unescaped_value =
+                    _ondemand_json_parser->unescape(value.get_raw_json_string(), unescape_buffer);
```

Comment:
If unescape_buffer is not big enough, will it cause buffer overflow?

Reply:
it's resized at least the size of original json